domain/src/test/kotlin/no/nav/su/se/bakover/domain/vedtak/VedtakTest.kt

```diff
@@ -129,7 +136,7 @@ internal class VedtakTest {
         val periode = år(2021)
         val bosituasjon = lagFullstendigBostiuasjon(periode)
 
-        val vedtak = lagVedtak(
+        val vedtak = lagVedtakInnvilgetSøknadsbehandling(
```

Comment:
Dubiøst for en funksjon som returnerer noe som overhodet ikke burde være innvilget.

Reply:
Jeg lager et et par tasks her.